packages/jest-validate/src/__tests__/validateCLIOptions.test.ts

```diff
@@ -9,57 +9,53 @@
 import validateCLIOptions from '../validateCLIOptions';
 
 test('validates yargs special options', () => {
-  const options = ['$0', '_', 'help', 'h'];
   const argv = {
-    $0: true,
-    _: true,
+    $0: 'foo',
+    _: ['bar'],
     h: false,
     help: false,
   };
-  expect(validateCLIOptions(argv, options)).toBe(true);
+
+  expect(validateCLIOptions(argv)).toBe(true);
```

Comment:
Strange thing. Second arg cannot be an array: 

https://github.com/facebook/jest/blob/ed0abb799c313f07563ad828a05299156bc88c54/packages/jest-validate/src/validateCLIOptions.ts#L64-L70

Reply:
The initial commit is here https://github.com/facebook/jest/pull/2613. Hm.. Still not clear what these arrays were supposed to do. Seems like nothing, or?